docs/reference/setup/logging-config.asciidoc

```diff
@@ -194,3 +196,36 @@ files (four rolled logs, and the active log).
 
 You can disable it in the `config/log4j2.properties` file by setting the deprecation
 log level to `error`.
+
+
+[float]
+[[json-logging]]
+=== Json log format
+
+To make parsing elasticsearch logs easier, logs are now printed in a json format.
+This is configured by a log4j layout property `appender.rolling.layout.type = ESJsonLayout`.
+This layout requires a type_name attribute to be set which is used to distinguish
+logs streams when parsing.
+[source,properties]
+--------------------------------------------------
+appender.rolling.layout.type = ESJsonLayout
+appender.rolling.layout.type_name = server
+--------------------------------------------------
+
+Each line contains a single json with set of attributes that are configured in `ESJsonLayout`.
+See this class javadoc for more details.
+When the log event contains a `Throwable`, json will be formatted multiline where first line contains
+all the regular attributes and subsequent lines will contain stacktrace formatted as a json array.
+
+
+NOTE: You can still use your own custom layout. To do that just replace line
```

Comment:
Scratch "just"? It does not convey any meaning here IMHO? Also: "replace line" -> replace *the* line"